src/router.js

```diff
@@ -126,10 +126,33 @@ const routes = [
     component: StatsView,
     beforeEnter: mustBeAuthenticated,
   },
+  // Catch every other unsupported route
+  {
+    path: '/:pathMatch(.*)*',
+    name: 'Not Found',
+    component: () => import(
+      /* webpackChunkName: "not-found-view" */
```

Comment:
Is this applicable to vite since it's rollup based? Looks like webpack config

Reply:
Nope, we’d need a plugin. Old habit :) — kind removing it? I have limited access today.

Reply:
```suggestion
```